src/NeoSharp.Core/Wallet/NEP6/NEP6WalletManager.cs

```diff
@@ -407,19 +409,13 @@ private IWalletAccount TryGetAccount(UInt160 accountScriptHash)
         }
 
         /// <summary>
-        /// Load a wallet using a fileInfo.
+        /// Load a wallet using a file name.
         /// </summary>
-        /// <param name="fileInfo">File info.</param>
-        public void Load(FileInfo fileInfo)
+        /// <param name="fileName">File name.</param>
+        public void Load(String fileName)
```

Comment:
always use "string" instead "String"
check the comments here

https://stackoverflow.com/questions/7074/what-is-the-difference-between-string-and-string-in-c